extensions/3DTILES_metadata/1.0.0/README.md

```diff
@@ -0,0 +1,1623 @@
+<!-- omit in toc -->
+# 3DTILES_metadata Extension
+
+**Version 1.0.0**, February 25, 2021
+
+<!-- omit in toc -->
+## Contributors
+
+* Sean Lilley, Cesium
+* Peter Gagliardi, Cesium
+* Sam Suhag, Cesium
+* Patrick Cozzi, Cesium
+* Bao Tran, Cesium
+
+<!-- omit in toc -->
+## Status
+
+Draft
+
+<!-- omit in toc -->
+## Dependencies
+
+Written against the 3D Tiles 1.0 specification.
+
+Adds new functionality to the [`3DTILES_implicit_tiling` extension](../../3DTILES_implicit_tiling/0.0.0). See [Implicit Tile Metadata](#implicit-tile-metadata).
+
+<!-- omit in toc -->
+## Optional vs. Required
+
+This extension is optional, meaning it should be placed in the tileset JSON `extensionsUsed` list, but not in the `extensionsRequired` list.
+
+<!-- omit in toc -->
+## Contents
+
+- [Overview](#overview)
+- [Use Cases](#use-cases)
+- [Compatibility Notes](#compatibility-notes)
+- [Concepts](#concepts)
+  - [Schemas](#schemas)
+  - [Tileset Metadata](#tileset-metadata)
+  - [Tile Metadata](#tile-metadata)
+    - [Implicit Tile Metadata](#implicit-tile-metadata)
+  - [Group Metadata](#group-metadata)
+  - [Feature Metadata](#feature-metadata)
+  - [Statistics](#statistics)
+  - [Semantics](#semantics)
+  - [Styling](#styling)
+- [JSON Schema Reference](#json-schema-reference)
+
+## Overview
+
+This extension defines a standard mechanism for adding metadata to 3D Tiles. This includes:
```

Comment:
Similar to the intro in #455, this should define a tad more what metadata is.

Readers will come into this with lots of ideas about what metadata means to them, e.g.,
* https://github.com/KhronosGroup/glTF/blob/master/extensions/2.0/Khronos/KHR_xmp/README.md#overview
* https://github.com/KhronosGroup/glTF/blob/master/extensions/2.0/Vendor/AGI_stk_metadata/README.md
* https://github.com/KhronosGroup/glTF/blob/master/extensions/2.0/Vendor/FB_geometry_metadata/README.md